packages/manager/modules/vps/src/dashboard/tile/index.js

```diff
@@ -0,0 +1,9 @@
+import angular from 'angular';
+
+import informationTile from './information';
+
+const moduleName = 'ovhManagerVpsDashboardTile';
+
+angular.module(moduleName, [informationTile]);
```

Comment:
No translations, no ui-kit, no ui-router ?

Reply:
No ui-router, as there is no routing set here.
No translations, as the child have it for itself.
Ok for ui-kit :)